packages/autorest.gotest/test/integrationtest/output/appplatform/armappplatform/__debug/go-tester-pre.yaml

```diff
@@ -40238,11 +40238,11 @@ language:
     name: AppPlatformManagementClient
     description: ''
     azureARM: true
-    containingModule: none
+    containingModule: ''
     generateDateTimeRFC3339Helper: true
     groupParameters: true
     headAsBoolean: true
-    module: github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/appplatform/armappplatform/v2
+    module: github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/appplatform/armappplatform
```

Comment:
Was moving back from /v2 intentional?

Reply:
This is just the intermediate metadata which has slightly changed. The end result is still the same, i.e. we generate the code with the `/v2` suffix as expected.